CHANGELOG.md

```diff
@@ -16,18 +16,51 @@
 
 ### Breaking changes
 
-* Lorem ipsum.
+* Disable copying of various classes to prevent incorrect use at compile time.
+* History type enumeration value `Replication::hist_Sync` renamed to
+  `Replication::hist_SyncClient`.
+* Bumps file format version from 6 to 7 due to addition of a 10th element into
+  `Group::m_top`. The new element is the history schema version, which is
+  crucial for managing the schema upgrade process of sync-type histories in a
+  way that is independent of core's Realm file format. The bump is necessary due
+  to lack of forwards compatibility. The changes are backwards compatible, and
+  automatic upgrade is implemented.
+* New pure virtual methods `get_history_schema_version()`,
+  `is_upgradable_history_schema()`, and `upgrade_history_schema()` in
+  `Replication` interface.
```

Comment:
I think we should add PR / issue links here. That would be a PR to `develop` and I acknowledge that this will delay release until tomorrow (morning). I still believe it is worth it as we have several breaking changes. Information about these should be readily available to all IMHO.